lib/components/Snackbar/__tests__/snackbar.test.tsx

```diff
@@ -0,0 +1,109 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { Snackbar } from "..";
+
+describe("Snackbar", () => {
+    it("renders with default props", () => {
+        const { container } = render(
+            <Snackbar
+                message="Test message"
```

Comment:
You can create a reusable object with props and pass it with spread syntax. Also, if value of the prop is true, it can be omitted.